README.md

```diff
@@ -264,11 +269,33 @@ We also recommend these lists.
 *Libraries for developing Web applications*
 
 * [Amon2](https://metacpan.org/pod/Amon2)
-* [Mojolicious](https://metacpan.org/pod/Mojolicious)
+* [Mojolicious](https://metacpan.org/pod/Mojolicious) - An all in one framework.
 * [Kossy](https://metacpan.org/pod/Kossy) - A Web framework with simple interface.
 * [Dancer](https://metacpan.org/pod/Dancer) ([Official site](http://perldancer.org/))
+* [Gantry](https://metacpan.org/pod/Gantry) - Web application framework for mod_perl, cgi, etc.
 * [Dancer2](https://metacpan.org/pod/Dancer2)
-* [Catalyst](https://metacpan.org/pod/Catalyst)
+* [Catalyst](https://metacpan.org/pod/Catalyst) - Overflowing with features. Very popular.
+
+
+## Web Framework-Like
+
+*Somewhere between templates and full on frameworks*
+
+* [Mason](https://metacpan.org/pod/Mason) - Powerful, high-performance templating for the web and beyond
+* [Poet](https://metacpan.org/pod/Poet) - a modern Perl web framework for Mason developers
+* [Embperl](https://metacpan.org/pod/Embperl) - Building dynamic Websites with Perl (sort of like Perl crossed with PHP)
+
+## Form Frameworks
+
+*Libraries that take the boredom & repetition out of (web and UI) forms*
+
+* [HTML::FormFu](https://metacpan.org/pod/HTML::FormFu) - HTML Form Creation, Rendering and Validation Framework
+* [HTML::FormFu::ExtJS](https://metacpan.org/pod/HTML::FormFu::ExtJS) - ExtJS form generation from HTML::FormFu config files
+* [Catalyst::Controller::HTML::FormFu](https://metacpan.org/pod/Catalyst-Controller-HTML-FormFu) - Use HTML::FormFu in Catalyst
+* [CGI::FormBuilder](https://metacpan.org/pod/CGI-FormBuilder) - Easily generate and process stateful forms
+* [Form::Toolkit](https://metacpan.org/pod/Form-Toolkit) - A toolkit to build Data centric Forms
+* [WWW::Form](https://metacpan.org/pod/WWW-Form) - Simple and extendable module that allows developers to handle HTML form input validation and display flexibly and consistently.
+* [Form::Sensible](https://metacpan.org/pod/Form-Sensible) - A sensible way to handle form based user interface
 
 ### Middlewares
```

Comment:
Indentation is different from the Table of Contents.
And, I think should be title change from "Middlewares", If that move here.